eg-next/examples/text-alignment.rs

```diff
@@ -0,0 +1,60 @@
+//! # Example: Text alignment
+//!
+//! Draw left/center/right aligned text in a containing box.
+
+use embedded_graphics::{
+    mono_font::{ascii::FONT_8X13, MonoTextStyleBuilder},
+    pixelcolor::Rgb888,
+    prelude::*,
+    text::{Alignment, Baseline, Text, TextStyleBuilder},
+};
+use embedded_graphics_simulator::{OutputSettingsBuilder, SimulatorDisplay, Window};
+
+fn main() -> Result<(), core::convert::Infallible> {
+    let mut display: SimulatorDisplay<Rgb888> = SimulatorDisplay::new(Size::new(512, 128));
+
+    let bounding_box = display.bounding_box();
+
+    let character_style = MonoTextStyleBuilder::new()
+        .font(&FONT_8X13)
+        .text_color(Rgb888::CSS_TOMATO)
+        .build();
+
+    Text::with_text_style(
+        "Left aligned text, origin top left",
+        bounding_box.top_left,
+        character_style,
+        TextStyleBuilder::new()
+            .alignment(Alignment::Left)
+            .baseline(Baseline::Top)
+            .build(),
```

Comment:
I would suggest to define all styles before drawing any text.

This is a general suggestions for all inline style builder calls. I've seen multiple examples where people have written wrappers around text drawing or other draw functions because our calls seem to be to complicated to use every time. But IMO many of these problems could have been solved by creating styles ahead of time.

Reply:
I think I've caught every instance of `TextStyleBuilder` and `PrimitiveStyleBuilder` and moved them to their own variables. Let me know if I missed one.